packages/discovery-provider/src/queries/generate_unpopulated_trending_tracks.py

```diff
@@ -128,14 +128,11 @@ def make_generate_unpopulated_trending(
     """Wraps a call for use in `use_redis_cache`, which
     expects to be passed a function with no arguments."""
```

Comment:
nite: comment no longer necessary... and actually... maybe this whole method no longer necessary...